js/tests/unit/button.js

```diff
@@ -110,6 +110,18 @@ $(function () {
     assert.ok($btn.hasClass('active'), 'btn has class active')
   })
 
+  QUnit.test('should prevent toggle active when disabled btn children are clicked', function (assert) {
+    assert.expect(2)
+    var $btn = $('<button class="btn disabled" data-toggle="button">mdo</button>')
```

Comment:
Could we please use [a checkbox/radio button group](http://getbootstrap.com/javascript/#buttons-checkbox-radio) here, since that's what the issue is about?